### DISCUSSION TITLE
Error at login

### DISCUSSION THREAD

**Discussion by chris199512** [bauerj/paperless_app#93] [Q&A]
![Screenshot_20220815-183137.jpg](https://user-images.githubusercontent.com/84684730/184723544-dbf284d2-a283-44a5-a304-12815207c427.jpg)
Hello, i have this error. On other discussion i see that an adblocker was the reason. I only use pihole. When i disable it, i get the error too. I use paperless ngx on debian 11 on proxmox.

**Comment by mvdkleijn** [comment]
Did you verify that you can reach the service from another computer?

You could use ping and/or telnet to check:

> ping ip address
> telnet ip address port

**Reply by chris199512** [reply]
![Screenshot_20220821-174250_Termux.jpg](https://user-images.githubusercontent.com/84684730/185800055-6665b50c-a0b1-4698-b5db-b6615c01088d.jpg)ping works from my smartphone. I dont set something special.

**Reply by bauerj** [reply]
More importantly: Can you open https://192.168.178.157:53918/ on your mobile browser?

**Reply by chris199512** [reply]
no. this port dont work. only 8000 works. What i have to do that it works?

**Reply by bauerj** [reply]
Then please enter https://192.168.178.157:8000/ as your URL.

**Reply by chris199512** [reply]
Thank you. This works. are there any limitations with it? Or does it work the same way?

**Reply by bauerj** [reply]
No, it should be all the same.
